src/misc/Header.tsx

```diff
@@ -15,17 +15,21 @@ const useStyles = makeStyles({
     },
     headerBar: {
         marginBottom: "10px",
-        display: "flex",
-        alignItems: "center",
-        flexDirection: "row",
-        padding: "0.5rem",
         margin: 0,
         width: "100%",
         background:
       "linear-gradient(90deg, rgba(154,12,52,1) 0%, rgba(0,0,0,1) 100%)",
         zIndex: 10,
         boxShadow: "rgba(0, 0, 0, 0.16) 0px 1px 4px",
     },
+    headerContainer: {
+        display: "flex",
+        alignItems: "center",
+        flexDirection: "row",
+        padding: "0.5rem",
+        maxWidth: "1080px",
```

Comment:
why add this?

Reply:
First 3 were in header bar before but just moved it onto the container and maxWidth just to stop the home and logout button being miles apart on widescreen